core/templates/pages/admin-page/roles-tab/admin-roles-tab.component.spec.ts

```diff
@@ -182,6 +190,20 @@ describe('Admin roles tab component ', function() {
       expect(component.managedTopicIds).toEqual(['topic_id_1']);
       expect(component.userIsBanned).toBeFalse();
     }));
+
+    it('should set status when viewUsersRoleAsync fails', fakeAsync(() => {
+      spyOn(adminBackendApiService, 'viewUsersRoleAsync').and.returnValue(
+        Promise.reject('Error')
+      );
+      spyOn(component.setStatusMessage, 'emit');
+      expect(component.roleIsCurrentlyBeingEdited).toBeFalse();
+
+      component.startEditing();
+      tick();
+
+      expect(component.roleIsCurrentlyBeingEdited).toBeFalse();
```

Comment:
Could you add a comment explaining why the value shouldn't change

Reply:
Done.